Gulpfile.js

```diff
@@ -0,0 +1,205 @@
+var gulp = require('gulp'),
+    plugins = require('gulp-load-plugins')();
+
+var del = require('del'),
+    path = require('path'),
+    merge = require('merge-stream'),
+    paths = require('./gulpfile.paths.js'),
+    recess = require('recess'),
+    Server = require('karma').Server,
+    pngquant = require('imagemin-pngquant'),
+    replace = require('gulp-replace'),
+    documentation = require('gulp-documentation'),
+    embedTemplates = require('gulp-angular-embed-templates'),
+    strip = require('gulp-strip-comments'),
+    flatten = require('gulp-flatten'),
```

Comment:
Tip: All modules starting with `gulp-` can be accessed via the `plugins` object.

Example:
`require('gulp-flatten') === plugins.flatten`